README.md

```diff
@@ -1,38 +1,82 @@
-# Newdle
+## WARNING: newdle is not yet production ready
+### Currently missing
+ * [ ] E-mail sending;
```

Comment:
can we skip those weird semicolons? :D

Reply:
And I'm pretty sure people won't update this list so I'd rather just mention it's still under heavy development and not ready for real usage.